src/nyc_trees/apps/login/forms.py

```diff
@@ -24,9 +24,11 @@
 
 
 class NycAuthenticationForm(AuthenticationForm):
+    username = forms.CharField(max_length=254,
+                               label="Username or Email Address")
 
     error_messages = {
-        'invalid_login': 'Please enter a correct username and password. '
+        'invalid_login': 'Please enter a correct username/email and password. '
                          'Note that password is case-sensitive',
```

Comment:
I suppose it should be email address, but otherwise yes. I feel like the slash could be interpreted as the user needed to enter both.